### ISSUE TITLE
Question - what made you quit spacemacs?

### ISSUE THREAD

**Issue by glynnforrest** [chrisbarrett/.emacs.d#1]
Your [spacemacs-layers](https://github.com/chrisbarrett/spacemacs-layers) repository has been a great reference for me while trying to switch to spacemacs from a home-grown config.

What made you abandon it and use this repo instead?

**Comment by chrisbarrett** [comment]
Someone else asked me that, so I'll post my response here.

tldr; Spacemacs is good, I just wanted more control of my config.

--

I think Spacemacs is one of the best ways to get up-and-running and comfortable with Emacs these days. I'm comfortable rolling my own config (that's how I learned) so I ended up wanting something tailored to how I work.

Spacemacs has a lot of infrastructure to make it easy for end users to customise, and comes with a lot of packages out of the box. But the tradeoff is that there's more code to work with when you're trying to debug, extend or bypass out-of-the-box behaviour. I only used a small subset of the features that Spacemacs provides, but I still found myself having to keep a close eye on their issue tracker to make sure their hacks didn't break my hacks. ;)

I've ended up porting some of the core utilities from Spacemacs so that I can still use their keybinding setup (which I think is really good), but writing everything else myself.

A related issue I had with my previous config was stability, especially since the package repo I used (MELPA) always build packages off master. I wanted to use git to manage my dependencies instead.

**Comment by chrisbarrett** [comment]
Plus, it was time for my annual .emacs.d spring cleaning & bankruptcy declaration anyway. :p

**Comment by glynnforrest** [comment]
Thanks for your detailed response; I agree with everything you've said here. I've gotten a little tired of trying to bypass what spacemacs does, so will be using that time to work on my own trimmed down config instead! I'm bringing the spacemacs keybindings with me though, they're very well thought out.
